### PR TITLE
Updated start script with managed cluster name

### PR DESCRIPTION
**Pull request by dislbenn**
Issue: https://github.com/open-cluster-management/backlog/issues/16685.
Signed-off-by: dislbenn <lavontae.bennett@gmail.com>

### CODE REVIEW

**Comment by dislbenn** [comment]
This PR will help our test select the correct managed cluster to test off of. CICD is providing a value within their `options.yaml` file, so this should help us select the correct one. If the value isn't there, we can select the managed cluster that starts with `canary-` if it is available.

**Comment by mqhackett** [comment]
The code looks goods.  The e2e test is failing, but once that passes it looks good to merge

**Review by mqhackett — APPROVED** [review]
Adds managed

**Review by jlpadilla — APPROVED** [review]
/lgtm

File: start-tests.sh [hunk 1]
```diff
@@ -35,26 +35,30 @@ fi
 
 # Load test config mounted at /resources/options.yaml
 OPTIONS_FILE=/resources/options.yaml
-USER_OPTIONS_FILE=./options.yaml
+USER_OPTIONS_FILE=./resources/options.yaml
```
Comment (jlpadilla) [inline start-tests.sh:38]: These files are different on purpose. First we try to read from `/resources/options.yaml` if the file doesn't exist then we read from `./options.yaml`

Comment (dislbenn) [reply start-tests.sh]: Reverted user `options.yaml` path to its original path.

**Review by jlpadilla — APPROVED** [review]
/lgtm

**Pull request merged**